apps/meteor/app/livechat/server/lib/LivechatTyped.ts

```diff
@@ -1,8 +1,16 @@
-import type { IOmnichannelRoom, IOmnichannelRoomClosingInfo, IUser, MessageTypesValues, ILivechatVisitor } from '@rocket.chat/core-typings';
+import type {
+	LivechatSendRequestData,
+	IOmnichannelRoom,
+	IOmnichannelRoomClosingInfo,
+	IUser,
+	MessageTypesValues,
+	ILivechatVisitor,
+} from '@rocket.chat/core-typings';
 import { isOmnichannelRoom } from '@rocket.chat/core-typings';
 import { LivechatDepartment, LivechatInquiry, LivechatRooms, Subscriptions, LivechatVisitors, Messages, Users } from '@rocket.chat/models';
 import moment from 'moment-timezone';
 import { TAPi18n } from 'meteor/rocketchat:tap-i18n';
+import { default as axios } from 'axios';
```

Comment:
`axios` is not one of our dependencies..

we should use our `fetch` implementation for HTTP requests from backend: https://github.com/RocketChat/Rocket.Chat/blob/258d82eafbb6085a4595993e8903f41d88468bcd/apps/meteor/server/lib/http/fetch.ts#L43

Reply:
As per our yesterday's discussion, removed axios package and added a new timeout mechanism to native fetch api